src/redux/selectors/orgUnitDistReport/getTransformedChartData.js

```diff
@@ -0,0 +1,80 @@
+import isEmpty from 'lodash.isempty'
+import i18n from '@dhis2/d2-i18n'
+import { getTitle, createGrid } from '../../../utils/dataTransformHelpers'
+import createDataTransformCache from '../../../utils/dataTransformCache'
+
+const FONT_FAMILY = "'Roboto', sans-serif"
```

Comment:
Although only used in this file, font and color IMO should be moved to either `src/config` or `src/utils` (preferably the former)